statsmodels/tsa/tests/test_stattools.py

```diff
@@ -602,6 +604,50 @@ def test_levinson_durbin_acov():
     assert_allclose(pacf, np.array([1, rho] + [0] * (m - 1)), atol=1e-8)
 
 
-if __name__=="__main__":
+@pytest.mark.parametrize("missing", ['conservative', 'drop', 'raise', 'none'])
+@pytest.mark.parametrize("fft", [False, True])
+@pytest.mark.parametrize("demean", [True, False])
+@pytest.mark.parametrize("unbiased", [True, False])
+def test_acovf_nlags(unbiased, demean, fft, missing):
+    rnd = np.random.RandomState(12345)
+    x = rnd.randn(250)
```

Comment:
my guess is that this would be better as a class with `x` created as a setup class
IIUC, we are creating the same randn a large number of times, both in this and the other test functions that are added here